swaps/rebalance.js

```diff
@@ -695,11 +695,12 @@ module.exports = (args, cbk) => {
               rebalance_fees_spent: tokAsBigTok(pay.fee),
             },
           ],
-        });
+        }
 
-        return cbk();
+        args.logger.info(rebalanceResult);
```

Comment:
This line should be unnecessary because the result will be logged already to info